src/sonic-yang-models/yang-models/sonic-event.yang

```diff
@@ -0,0 +1,134 @@
+  module sonic-event {
```

Comment:
Would you add a code to call this event from one of the App to showcase the usage?